components/elm/docs/user-guide/index.md

```diff
@@ -1 +1,89 @@
-start of the ELM User's Guide
+# ELM User's Guide
+
+This User's Guide describes how to set up and run ELM.
+
+## Table of Conents
```

Comment:
Remove this. A TOC is auto-generated on the right-hand-side of the page.

Reply:
Done.